src/NinjaTools.FluentMockServer.API/Proxy/Evaluation/Visitors/IPartialVisitor.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+using NinjaTools.FluentMockServer.API.Models;
+
+namespace NinjaTools.FluentMockServer.API.Proxy.Evaluation.Visitors
+{
+    public interface IPartialVisitor : IVisitor
+    {
+        void VisitCookies(IDictionary<string, string>? cookies);
+        void VisitHeaders(IDictionary<string, string[]>? headers);
+        void VisitPath(string? path);
+        void VisitMethod(string? method);
+        void VisitQuery(string? query);
+
+        void VisitBody(RequestBodyMatcher? bodyMatcher);// => VisitBody(bodyMatcher.Content, bodyMatcher.MatchExact, bodyMatcher.Type);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)